src/dbmanager.cpp

```diff
@@ -57,6 +57,33 @@ bool DBManager::isNoteExist(NoteData* note)
     return query.value(0).toInt() == 1;
 }
 
+NoteData* DBManager::getNote(QString id) {
+    QSqlQuery query;
+
+    int parsedId = id.split('_')[1].toInt();
+    QString queryStr = QStringLiteral("SELECT * FROM active_notes WHERE id = %1 LIMIT 1").arg(parsedId);
+    query.exec(queryStr);
+
+    if (query.first()) {
+        NoteData* note = new NoteData();
```

Comment:
this function might be used elsewhere, so I think it's better to set the parent "this", so if it's not deleted explicitly, we are sure that there would be no leak ;)